plugin/action/throttle/throttle.go

```diff
@@ -338,6 +354,13 @@ func (p *Plugin) isAllowed(event *pipeline.Event) bool {
 			throttleKey = val
 		}
 	}
+	keyLimitOverride := ""
+	if len(p.config.RedisBackendCfg.LimiterKeyField_) > 0 {
+		val := event.Root.Dig(p.config.RedisBackendCfg.LimiterKeyField_...).AsString()
+		if val != "" {
+			keyLimitOverride = val
+		}
+	}
```

Comment:
may be `keyLimitOverride := event.Root.Dig(p.config.RedisBackendCfg.LimiterKeyField_...).AsString()`

Reply:
Done